great-idea/css/index.css

```diff
@@ -62,3 +62,141 @@ h1, h2, h3, h4, h5 {
     letter-spacing: 1px;
     margin-bottom: 15px;
 }
+
+
+/* ~~~~~~~~~~~~~~~ My Code ~~~~~~~~~~~~~~~ */
+
+.container {
+	display: flex;
+	
+	margin-top: 20px;
+	margin-left: 40px;
+	margin-right: 40px;
+	margin-bottom: 60px;
+
+	flex-direction: column;
+}
+
+/* ~~~~~~~~~~ Header ~~~~~~~~~~ */
+
+header {
+	display: flex;
+	justify-content: space-between;
+	align-content: center;
+	/* background: lightblue; */
+}
+
+header nav {
+	display: flex;
+	width: 75%;
+	justify-content: space-between;
+	align-content: center;
+}
+
+header nav a {
+	display: flex;
+	text-decoration: none;
+	color: grey;
+	align-self: center;
+}
+
+/* ~~~~~~~~~~ CTA ~~~~~~~~~~ */
+
+.cta {
+	display: flex;
+	margin-top: 80px;
+	justify-content: space-around;
+}
+
+.cta .cta-text {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+}
+
+.cta .cta-text h1 {
+	display: flex;
+	font-size: 70px;
+	text-align: center;
+	/* background: lightblue; */
+}
+
+.cta .cta-text button {
+	margin-left: 60px;
+	margin-right: 60px;
+	padding-top: 4px;
+	padding-bottom: 4px;
+
```

Comment:
For readability, I recommend eliminating blank lines of code within a block unless there's a compelling reason to have them.